app/assets/stylesheets/layout.scss

```diff
@@ -39,14 +39,20 @@ main {
   padding-bottom: 1rem;
   color: #ccc;
   background-color: #2f3232;
+
+  .container {
+    @media screen and (min-width: 40em) {
+      padding: 0 20px;
+    }
+  }
 }
 
 .site-logo {
   width: 188px;
   height: 20px;
   margin-top: 0.3rem;
 
-  @media (max-width: 34em) {
+  @media screen and (max-width: 40em) {
```

Comment:
@tarebyte I've added this in mainly by habit, but by adding `screen`, the `@media` query is simply more descriptive as it is now specified that these styles are for screens and not the other media types defined in the CSS3 specification, such as `print` or `speech`. 

If you were to print out a webpage, for example, then the navigation becomes useless therefore you can specify a media query to remove the navigation when the site is printed, like this:

``` css
@media print {
    nav { display: none; }
}
```